src/main/java/com/thebois/views/GameScreen.java

```diff
@@ -22,31 +23,34 @@
     private final OrthographicCamera camera;
     private final FitViewport viewport;
     private final Color backgroundColor = new Color(0, 0, 0.2f, 1);
-    private final Skin skin;
-    private final GameView gameView;
+    private final Actor gameView;
     private final InfoView infoView;
+    private final ToolbarView toolbarView;
```

Comment:
If you're making `gameView` an `Actor` then I think all three of them should be `Actor`s.

Reply:
At the moment the other 2 are not actors while `GameView` has always been an actor. I guess that the other 2 could be made into `IActorViews` if we want that. Or I change `gameView` back to `GameView`

Reply:
Ah, they should all ideally be extending `Actor`, but that's another refactor so let's just leave it as it is right now.